bindings/nodejs/hawktracer_client_nodejs.cpp

```diff
@@ -37,16 +38,11 @@ Value Client::start(const CallbackInfo& info)
         ClientContext::create(
             _source,
             _maps,
+            _wait_for_server,
             [this]()
             {
                 _notify_new_event();
             }));
-    if (_state.is_started()) {
-        Ref();
```

Comment:
This still needs to be called after it's started. This prevents from the Client object from being garbage collected.

Reply:
Done